distributing/index.po

```diff
@@ -85,7 +86,6 @@ msgstr ""
 "Python"
 
 #: ../Doc/distributing/index.rst:37
-#, fuzzy
```

Comment:
Some fuzzy string are really hard to spot with naked eyes, sometime you have to check upstream what changed. Here's what changed: https://github.com/python/cpython/commit/d34ac305327420bd68f05e201b63d2a665b073f8

I found it locally on a cpython clone using `git log -p Doc/distributing/index.rst`.

So only the 2nd `b` of `Bitbucket` changed, so you should change it too.

It was not an easy one to start with :]